pkg/cmd/audit/io.go

```diff
@@ -122,7 +124,11 @@ func PrintTopByUserAuditEvents(writer io.Writer, numToDisplay int, events []*aud
 	countUsers := map[string][]*auditv1.Event{}
 
 	for _, event := range events {
-		countUsers[event.User.Username] = append(countUsers[event.User.Username], event)
+		username := event.User.Username
+		if fieldManager := QueryParams(event.RequestURI).Get("fieldManager"); len(fieldManager) > 0 {
+			username = fmt.Sprintf("%s[%s]", username, fieldManager)
```

Comment:
won't this be also applied even if `field-manager` wasn't specified ?

Reply:
I think if the `if fieldManager := QueryParams(event.RequestURI).Get("fieldManager"); len(fieldManager) > 0 {` works as I expect, this only shows up if the fieldManager is present.